api/src/main/java/org/apache/iceberg/ContentFile.java

```diff
@@ -141,4 +149,5 @@
    *         null value counts, or nan value counts
    */
   F copyWithoutStats();
+
```

Comment:
Nit: Can you remove this whitespace change? It isn't needed and could cause conflicts.

Reply:
Nevermind, this is the only issue and it isn't worth blocking.